.github/ISSUE_TEMPLATE/early-design-review.md

```diff
@@ -0,0 +1,53 @@
+---
```

Comment:
So while github's wizard doesn't let you control it... the filenames here control the order [the options](https://github.com/w3ctag/design-reviews/issues/new/choose) are given in.

I think this should probably be before the specification review one, though I don't feel strongly.  If you agree, it should probably be prefixed with `005-`, if not, with `015-`, so that it sorts appropriately.

Reply:
I agree, and will rename to `005-*`.